services/rest/service.go

```diff
@@ -52,14 +37,35 @@ func (s Service) Init(ctx context.Context) error {
 		Handler: router,
 		Addr:    ":" + config.ServerWs.AppPort,
 	}
-	s.s = server
-	return server.ListenAndServe()
+	return &Service{
+		s:         server,
+		Collector: c,
+	}
+}
+
+func pingHandler(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusOK)
+	w.Write([]byte("pong"))
+}
+
+func reportConnectionMetrics(conn connection.Table) {
+	t := time.Tick(config.MetricStatsd.FlushPeriodMs)
+	for {
+		<-t
+		for k, v := range conn.TotalConnectionPerGroup() {
+			metrics.Gauge("connections_count_current", v, fmt.Sprintf("conn_group=%s", k))
+		}
+	}
+}
+
+func (s *Service) Init(ctx context.Context) error {
+	return s.s.ListenAndServe()
 }
 
-func (s Service) Name() string {
+func (s *Service) Name() string {
```

Comment:
`s` is unused and can be removed

Reply:
Same, this is an interface implementation.